RebornCore/src/main/java/reborncore/common/recipes/IUpgradeHandler.java

```diff
@@ -39,7 +39,7 @@
 
 	double getPowerMultiplier();
 
-	double getEuPerTick(double baseEu);
+	long getEuPerTick(long baseEu);
```

Comment:
Not really something to worry too much here, still funny to see "EU" refrences around the place. At somepoint we should clean this stuff up.